js/symbol/shaping.js

```diff
@@ -24,6 +24,8 @@ function Shaping(positionedGlyphs, text, top, bottom, left, right) {
     this.right = right;
 }
 
+var newLine = 0x0a;
```

Comment:
What about carriage returns? Can we be sure that Windows-style CRLF won’t occur?